playground/src/components/custom-views/constants.js

```diff
@@ -0,0 +1,27 @@
+import {
+  TCustomViewSize,
+  TCustomViewType,
+} from '@commercetools-frontend/application-shell';
```

Comment:
```suggestion
import type {
  TCustomViewSize,
  TCustomViewType,
} from '@commercetools-frontend/application-shell';
```

Reply:
These are enums...actually let me remove these exports, we don't really need them. https://github.com/commercetools/merchant-center-application-kit/pull/3175/commits/4894226190ea078a6740d2adf46e5ecc02ab6ea1